components/nif/nifstream.cpp

```diff
@@ -24,4 +24,21 @@ namespace Nif
         t.scale = getFloat();
         return t;
     }
+
+    ///Currently specific for 4.0.0.2 and earlier
+    bool NIFStream::getBoolean()
+    {
+        return !!getInt();
```

Comment:
Why `!` is used twice?

Reply:
This is one of the explicit ways to avoid int-to-bool implicit conversion warnings which are currently silenced. Another way is a comparison to zero. !!getInt() is a boolean (not an integer) expression which means that the integer is not equal to zero. I didn't change it to anything because that's the way it was in the NIF code already.

Reply:
Are there more readable ways to avoid such warnings?